elide-core/src/main/java/com/yahoo/elide/jsonapi/models/Resource.java

```diff
@@ -146,13 +148,20 @@ public boolean equals(Object obj) {
 
     public PersistentResource<?> toPersistentResource(RequestScope requestScope)
         throws ForbiddenAccessException, InvalidObjectIdentifierException {
-        Class<?> cls = requestScope.getDictionary().getEntityClass(type);
+        EntityDictionary dictionary = requestScope.getDictionary();
+        Class<?> cls = dictionary.getEntityClass(type);
+
         if (cls == null) {
             throw new UnknownEntityException(type);
         }
         if (id == null) {
             throw new InvalidObjectIdentifierException(id, type);
         }
-        return PersistentResource.loadRecord(cls, id, requestScope);
+
+        EntityProjection projection = EntityProjection.builder()
```

Comment:
Do we need to build a new projection every time we use this method?

Reply:
Where would it come from?  This is not tied directly to the request path.  This typically comes from the JSON body.  There is no root projection in that case.  The object is just be loaded by its ID.